lib/osf-components/addon/components/draft-registration-card/template.hbs

```diff
@@ -32,6 +32,10 @@
                     {{t 'osf-components.draft-registration-card.form_type'}}
                     {{this.draftRegistration.registrationSchema.name}}
                 </p>
+                <p data-test-provider-name>
+                    {{t 'osf-components.draft-registration-card.provider'}}
+                    {{this.draftRegistration.provider.name}}
```

Comment:
Is this how we want to show the provider? I'd always imagined, though I can't say if I'd seen this mocked up anywhere, that we'd put the provider icon in the upper-right section of the card. (Checks My Registration Page documentation) It is how we want to do this. I still think my way would be nice. :)

Reply:
I vaguely recall the idea of showing the provider logo somewhere on the card, but am struggling to find that mockup right now, but I definitely recall asking Mark if just showing the name would suffice for the first cut, just in case the logo became a bit of a rabbit hole. I can have a crack at showing the logo as right now the registry name is buried and hard to see.